lib/krane/cli/deploy_command.rb

```diff
@@ -10,10 +10,7 @@ class DeployCommand
         kube-public
       )
       OPTIONS = {
-        "bindings" => { type: :array, banner: "foo=bar abc=def",
-                        desc: "Expose additional variables to ERB templates (format: k1=v1 k2=v2, JSON string or file "\
-                          "(JSON or YAML) path prefixed by '@')" },
-        "filenames" => { type: :array, banner: 'config/deploy/production config/deploy/my-extra-resource.yml',
+        "filenames" => { type: :string, banner: 'config/deploy/production config/deploy/my-extra-resource.yml',
```

Comment:
We didn't have a black box test for parsing `-f -`. Turns out that array input type can't have leading `-` or thor assumes its the start of a new flag :(